controllers/snapshot/snapshot_adapter.go

```diff
@@ -195,7 +195,7 @@ func (a *Adapter) EnsureSnapshotEnvironmentBindingExist() (results.OperationResu
 		return results.ContinueProcessing()
 	}
 
-	if !gitops.IsSnapshotCreatedByPushEvent(a.snapshot) {
+	if gitops.IsSnapshotCreatedByPACPullRequestEvent(a.snapshot) {
 		a.logger.Info("The Snapshot won't be deployed because it's not created by a push event.")
```

Comment:
Maybe we can also change this message to say:
```
The Snapshot won't be deployed because it was created for a PaC pull request event.
```

Reply:
Thanks! Updated as you suggested.